first_game.py

```diff
@@ -0,0 +1,42 @@
+print("You're a passenger of a ship that sank near a beautiful tropical island. After reaching the beach you look around and decide on one of three options. Type 1 if you 'set a fireplace', type 2 if you 'sit down and cry', type 3 if you 'go to the jungle to get some food'")
```

Comment:
Use multiline """abc""" strings to separate lines longer than 80.

Reply:
Try and not use single quotes inside double quotes (like "abc 'abc' abc"), it can confuse the python interpreter sometimes.

If you need to use a quotation sign you can use so-called Escape Sequences. If you put a backslash in front of a quotation mark inside a string, this will tell Python interpreter to ignore any "special" meaning that a sign can have.

For example "abc \'abc\' abc" is better than "abc 'abc' abc", even if you will see no apparent gain.

More here: https://www.python-ds.com/python-3-escape-sequences



https://www.python-ds.com/python-3-escape-sequences